build.gradle

```
@@ -38,8 +39,92 @@ tasks.named('bootBuildImage') {
 
 tasks.named('test') {
     useJUnitPlatform()
+    finalizedBy 'jacocoTestReport'
 }
 
+jacoco {
+    toolVersion = "0.8.8"
+}
+
+def jacocoDir = "$buildDir/reports/"
```

`$buildDir` 접근이 Gradle 9버전에서 부터 Deprecated 된다고 하네용?
아직 8.4버전을 사용중이긴 한데, 바꾸는것도 괜찮아보입니다!
```suggestion
def jacocoDir = layout.buildDirectory.dir("reports/")
```